src/components/PlayerInfo/PlayerComponents.test.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
```

Comment:
Can this be split up so that each component has it's own test file?

Reply:
Was thinking that, but thought the components are kinda related, so decided to have the tests in the same file. But I'll split it now.